adze-web/src/state/manifestSlice.js

```diff
@@ -55,6 +55,7 @@ export const addLinkByUrl = createAsyncThunk(
      title: linkDesc.title,
      url: linkDesc.url,
      timestamp_ms: new Date().getTime(),
+     tags: linkAndCreds.tags

```

Comment:
We should change `linkAndCreds` to be more accurate, since now we are taking in the link, the credentials, and a list of tags to apply.